### ISSUE TITLE
Safari 6

### ISSUE THREAD

**Issue by alexgodlex** [kylefox/Typekit-Tweaker#6]
Seems its not working with Safari 6.1

**Comment by kylefox** [comment]
Yeah - it's actually due to Typekit changing the markup of their page slightly, which breaks Typekit Tweaker.

I've been in touch with Typekit to see if there's anyway they can add the "specimen-editor" CSS class back to the page -- doing so would magically unbreak the extension.
